src/lib/parserRaw.c

```diff
@@ -680,9 +679,9 @@ RdbStatus elementRawModule(RdbParser *p) {
                     break;
                 }
 
-                /* No new-key precedes module aux. Init Aggregator of bulks here.
-                 * Note that the call is made from a safe state */
-                aggAllocFirstBulk(p);
+                /* No new-key precedes module aux. Init Aggregator of bulks here. */
+                aggReset(p);
+
                 updateElementState(p, ST_AUX_START, 0);
```

Comment:
personally i think it's odd that this code is just placed outside the `RDB_TYPE_MODULE_2` condition.
i think it should be an `else` with separate scope, and a comment stating that it belongs to `RDB_OPCODE_MODULE_AUX`

Reply:
Done. Thanks.